### DISCUSSION TITLE
Add a confirmation dialog before removing custom permission settings

### DISCUSSION THREAD

**Discussion by omusil24** [directus/directus#15953] [Feature Requests]
In case you have a bunch of custom permissions set for a role+collection+action, and you want to change them or add a new one for example, it's easy to accidentally click on **All Access** or **No Access** in the popup menu and thereby delete all those custom permissions.

It would be good if there was a confirmation dialog asking the user if they really want to remove all the custom permissions, in case they clicked **All Access** or **No Access** accidentally.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
